### DISCUSSION TITLE
(sidebar) am I missing additional styles?

### DISCUSSION THREAD

**Discussion by thisisfel1x** [shadcn-ui/ui#5546] [Q&A]
Just wanted to try the newly added sidebar components.... so I copied for testing purposes the folder sidebar-07 in the github repository.

In comparison to the blocks page on the official website, my version lacks, maybe in styling? Icons to big and black, and block is not on the same height.

Have I missed something? My globals.css is as good as empty. Glad for help :D

![grafik](https://github.com/user-attachments/assets/beedaba1-bfe3-4066-948a-7fa6f5270f56)

**Comment by krish-kant** [comment]
@layer base {
  :root {
    --sidebar-background: 0 0% 98%;
    --sidebar-foreground: 240 5.3% 26.1%;
    --sidebar-primary: 240 5.9% 10%;
    --sidebar-primary-foreground: 0 0% 98%;
    --sidebar-accent: 240 4.8% 95.9%;
    --sidebar-accent-foreground: 240 5.9% 10%;
    --sidebar-border: 220 13% 91%;
    --sidebar-ring: 217.2 91.2% 59.8%;
  }

  .dark {
    --sidebar-background: 240 5.9% 10%;
    --sidebar-foreground: 240 4.8% 95.9%;
    --sidebar-primary: 224.3 76.3% 48%;
    --sidebar-primary-foreground: 0 0% 100%;
    --sidebar-accent: 240 3.7% 15.9%;
    --sidebar-accent-foreground: 240 4.8% 95.9%;
    --sidebar-border: 240 3.7% 15.9%;
    --sidebar-ring: 217.2 91.2% 59.8%;
  }
}

**Comment by Raza9798** [comment]
![image](https://github.com/user-attachments/assets/52ed6890-da49-4cd4-9860-81240edd6714)


IN the darkmode it renders as above @krish-kant
